### ISSUE TITLE
no 'limits.h', 'sys/types.h' and "recipe for target 'example.o' failed"

### ISSUE THREAD

**Issue by YezCohen** [deptofdefense/AndroidTacticalAssaultKit-CIV#328]
Hi there,
I'm new to this project, and trying to create the developer environment just so i could compile the app to an .apk file to run on my phone before starting to play with it.

While going over the instructions to build the project on in "[BUILDING.md](https://github.com/deptofdefense/AndroidTacticalAssaultKit-CIV/blob/master/BUILDING.md)", i got to step 3 and runing the second line in the terminal:
`make TARGET=android-armeabi-v7a GDAL_USE_KDU=no build_spatialite build_gdal build_commoncommo build_assimp`
it runs, and then gets an error, this is what i got:

> make -C /home/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-armeabi-v7a-release/zlib
> make[1]: Entering directory '/home/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-armeabi-v7a-release/zlib'
> /home/j/ndk/android-ndk-r12b/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/bin/arm-linux-androideabi-gcc -fstack-protector-all -mfloat-abi=softfp -mfpu=neon -march=armv7-a -mtune=cortex-a8 -ftree-vectorize -O3 -DNO_SIZE_T= -DNO_FSEEKO -DNO_STRERROR -DNO_snprintf -DHAS_sprintf_void -I. -c -o example.o test/example.c
> In file included from /home/j/ndk/android-ndk-r12b/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/include-fixed/syslimits.h:7:0,
>                  from /home/j/ndk/android-ndk-r12b/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/include-fixed/limits.h:34,
>                  from ./zconf.h:418,
>                  from ./zlib.h:34,
>                  from test/example.c:8:
> /home/j/ndk/android-ndk-r12b/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/lib/gcc/arm-linux-androideabi/4.9.x/include-fixed/limits.h:168:61: error: no include path in which to search for limits.h
>  #include_next <limits.h>  /* recurse down to the real one */
>                                                              ^
> In file included from ./zlib.h:34:0,
>                  from test/example.c:8:
> ./zconf.h:444:48: fatal error: sys/types.h: No such file or directory
>      include <sys/types.h>      /* for off_t */
>                                                 ^
> compilation terminated.
> Makefile:147: recipe for target 'example.o' failed
> make[1]: *** [example.o] Error 1
> make[1]: Leaving directory '/home/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-armeabi-v7a-release/zlib'
> mk/zlib.mk:32: recipe for target 'zlib_build' failed
> make: *** [zlib_build] Error 2

does anyone have any idea how to fix it?

Any help would be much appreciated!

**Comment by talentedbrute** [comment]
Is there a reason you are trying to build ancient baseline?  This GitHub project has been abandoned and you can get the latest SDKs for development by registering for an account on tak.gov.

**Comment by AIT-tommy** [comment]
> Is there a reason you are trying to build ancient baseline? This GitHub project has been abandoned and you can get the latest SDKs for development by registering for an account on tak.gov.

@talentedbrute 
TAK.gov points to this repository. Am I missing something on getting the most up to date codebase for TAK?

**Comment by talentedbrute** [comment]
Yeah don't use this project, it's practically abandoned at this point.  If you are looking for SDKs, as I stated before, get a TAK.gov account.  Using this source is useless and doesn't gain you anything.  What is on GitHub is source for v4.6 the latest release of the app is v5.1 on TAK.gov.
